src/wagtailbakery/api_views.py

```diff
@@ -1,16 +1,20 @@
+import os
 import json
 import logging
-import os
 
-from bakery.views import BuildableMixin
 from django.conf import settings
-from django.contrib.contenttypes.models import ContentType
-from wagtail.api.v2.endpoints import PagesAPIEndpoint
-from wagtail.api.v2.router import WagtailAPIRouter
+from bakery.views import BuildableMixin
+
 from wagtail.core.models import Page, Site
+from wagtail.api.v2.router import WagtailAPIRouter
+from wagtail.api.v2.endpoints import PagesAPIEndpoint
+
+from django.contrib.contenttypes.models import ContentType
 
 logger = logging.getLogger(__name__)
 
+BAKERY_RESULTS_PER_PAGE = getattr(settings, 'BAKERY_RESULTS_PER_PAGE', 25)
```

Comment:
Changing from the default 20 results in a `limit cannot be higher than 20` API error.

Reply:
yep noticed already committed but not pushed